src/Yoti.Auth/DigitalIdentity/GetSessionResult.cs

```diff
@@ -1,48 +1,32 @@
-﻿using Newtonsoft.Json;
-
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+using Yoti.Auth.DigitalIdentity.Extensions;
 
 namespace Yoti.Auth.DigitalIdentity
 {
-    public class GetSessionResult
+    public class GetQrCodeResult
```

Comment:
in here the file name is `GetSessionResult.cs` and the class name `GetQrCodeResult`

Reply:
File names were mixed and corrected now